plugins/woocommerce/client/legacy/js/admin/meta-boxes-product.js

```diff
@@ -902,6 +936,34 @@ jQuery( function ( $ ) {
 		}
 	);
 
+	// Go to variations tab when clicking on link in the general tab message
+	$( document.body ).on(
```

Comment:
Can we use something more specific, like the div `woocommerce-product-data`?

The code would look like this:
```
$( '#woocommerce-product-data' ).on(
```